accounting/overview/getting_started/setup.rst

```diff
@@ -1,36 +1,194 @@
-5.  Setting up any foreign currencies
-6.  Importing your customers
-7.  Importing your suppliers
-8.  Importing your products
-9.  Importing your outstanding transactions
-10. Importing your starting balances
-11. Define the users for accounting
-
-.. image:: ./media/setup02.png
+.. note::
+   :doc:`Click here <../../bank/feeds/bank_synchronization>` for more information about this feature.
+
+If your bank can’t be synchronized automatically, or if you prefer not to sync your bank account
+with your database, you may also configure your bank account manually by clicking on **Create it**,
+and filling out the form.
+
+- **Name**: The bank account's name, as displayed on Odoo.
+- **Account Number**: Your bank account number (IBAN in Europe).
+- **Bank**: Click on *Create and Edit* to configure the bank's details. Add the real bank's name
+  and its Identifier Code (BIC or SWIFT)
+- **Code**: This code is your Journal's *Short Code*, as displayed on Odoo. By default, Odoo creates
+  a new Journal with this Short Code.
+- **Journal**: If this field is displayed, then select the *Journal* you want to use to record the
+  financial transactions linked to this bank account or create a new one by clicking on *Create and
+  Edit*.
+
+.. note::
+   You can add as many bank accounts as needed with this tool by going to :menuselection:`Accounting
+   --> Configuration`, and clicking on **Add a Bank Account**.
+
+.. _accounting-setup-periods:
+
+Accounting Periods
+------------------
+
+Define here your **Fiscal Years**’ opening and closing dates, which are used to generate reports
+automatically, and your **Tax Return Periodicity**, along with a reminder to never miss a tax return
+deadline.
+
+By default, the opening date is set on the 1st of January and the closing date on the 31st of
+December, as this is the most common use.
+
+.. note::
+   You can also change these settings by going to :menuselection:`Accounting --> Configuration -->
+   Settings --> Fiscal Periods` and updating the values.
+
+.. _accounting-setup-chart:
+
+Chart of Accounts
+-----------------
+
+With this menu, you can add accounts to your **Chart of Accounts** and indicate their initial
+opening balances.
+
+Basic settings are displayed on this page to help you review your Chart of Accounts. To access all
+the settings of an account, click on the *double arrow button* at the end of the line.
+
+.. image:: media/setup_chart_of_accounts.png
    :align: center
+   :alt: Setup of the Chart of Accounts and their opening balances in Odoo Accounting
+
+.. note::
+   :doc:`Click here <chart_of_accounts>` for more information on how to configure your Chart of
+   Accounts.
+
+Invoicing onboarding banner
+===========================
+
+There is another **step-by-step onboarding banner** that helps you take advantage of your Odoo
+Invoicing and Accounting apps.
+
+The **Invoicing onboarding banner** is the one that welcomes you if you use the Invoicing app rather
```

Comment:
```suggestion
| The *Invoicing onboarding banner* is the one that welcomes you if you use the Invoicing app rather
```

Reply:
I was told to break lines but not paragraphs, and if I'm not mistaken, I was also told that is not necessary to leave blanc lines (I could be wrong here, I don't clearly remember).
-> https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#break-the-line-but-not-the-paragraph

This is my last comment. I think I'm done then!